Dependency of dependency causing WARNINGS at compile time

When using protoc-gen-gogo I see messages like this for depenencies of a dependency:

```
2016/10/03 12:13:29 protoc-gen-gogo: WARNING: failed finding publicly imported dependency for .<package_message>, used in <my_package>/<my_package>.proto
```

When explicitly importing the package in the affected protofile, this warning disappears, but obviously I then instead get the warning that there are unused imports. This seems to imply that there is a problem with the dependency-of-dependency resolution.

Sure! See below (the `from ...` is the `*o.File().Name`).

```
panic: protoc-gen-gogo: WARNING: failed finding publicly imported dependency for .cockroach.util.hlc.Timestamp, used in cockroach/pkg/storage/engine/enginepb/mvcc.proto, from cockroach/pkg/util/hlc/timestamp.proto

goroutine 1 [running]:
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).ObjectNamed(0xc4200aa900, 0xc4202b25a0, 0x1d, 0x10, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1130 +0x5fa
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).IsMap(0xc4200aa900, 0xc4202b4a10, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/helper.go:219 +0x61
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc4202b4a10, 0xc4204fd1d0, 0x2, 0x2, 0xc4201b6628, 0x0, 0x0, 0xb)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:449 +0x5f
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc42038b6c0, 0xc4204fd1d0, 0x2, 0x2, 0xc4201b6628, 0x0, 0x0, 0xc4201b6668)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:474 +0x522
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).loops(0xc42007e1e0, 0xc42038b6c0, 0xc4204d0c80, 0xc4208dafc0)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:490 +0xb2
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).Generate(0xc42007e1e0, 0xc4200a8870)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:525 +0x4fb
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).runPlugins(0xc4200aa900, 0xc4200a8870)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1226 +0x59
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).generate(0xc4200aa900, 0xc4200a8870)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1282 +0x377
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).GenerateAllFiles(0xc4200aa900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1212 +0x259
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command.Generate(0xc4200540c0, 0x1e)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command/command.go:131 +0x232
main.main()
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/pkg/cmd/protoc-gen-gogoroach/main.go:91 +0x162
--gogoroach_out: protoc-gen-gogoroach: Plugin failed with status code 2.

panic: protoc-gen-gogo: WARNING: failed finding publicly imported dependency for .cockroach.util.hlc.Timestamp, used in cockroach/pkg/storage/engine/enginepb/mvcc.proto, from cockroach/pkg/util/hlc/timestamp.proto

goroutine 1 [running]:
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).ObjectNamed(0xc4200abc00, 0xc42021f800, 0x1d, 0xc4200e8448, 0x1011dfd)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1130 +0x5fa
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).IsMap(0xc4200abc00, 0xc420227260, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/helper.go:219 +0x61
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc420227260, 0xc4203782b0, 0x2, 0x2, 0xc4200e8628, 0x0, 0x0, 0xb)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:449 +0x5f
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc4202c4e70, 0xc4203782b0, 0x2, 0x2, 0xc4200e8628, 0x0, 0x0, 0xc4200e8668)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:474 +0x522
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).loops(0xc42007e1e0, 0xc4202c4e70, 0xc420342f00, 0xc4204e2a00)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:490 +0xb2
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).Generate(0xc42007e1e0, 0xc4200a8900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:525 +0x4fb
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).runPlugins(0xc4200abc00, 0xc4200a8900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1226 +0x59
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).generate(0xc4200abc00, 0xc4200a8900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1282 +0x377
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).GenerateAllFiles(0xc4200abc00)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1212 +0x259
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command.Generate(0xc4200540c0, 0x12)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command/command.go:131 +0x232
main.main()
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/pkg/cmd/protoc-gen-gogoroach/main.go:91 +0x162
--gogoroach_out: protoc-gen-gogoroach: Plugin failed with status code 2.
panic: protoc-gen-gogo: WARNING: failed finding publicly imported dependency for .cockroach.util.hlc.Timestamp, used in cockroach/pkg/storage/engine/enginepb/mvcc.proto, from cockroach/pkg/util/hlc/timestamp.proto

goroutine 1 [running]:
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).ObjectNamed(0xc4200aac00, 0xc4201a03e0, 0x1d, 0x10, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1130 +0x5fa
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).IsMap(0xc4200aac00, 0xc42019f960, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/helper.go:219 +0x61
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc42019f960, 0xc42039caa0, 0x2, 0x2, 0xc4203b8628, 0x0, 0x0, 0xb)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:449 +0x5f
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc4201b0930, 0xc42039caa0, 0x2, 0x2, 0xc4203b8628, 0x0, 0x0, 0xc4203b8668)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:474 +0x522
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).loops(0xc42007e1e0, 0xc4201b0930, 0xc4201d0320, 0xc420397a40)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:490 +0xb2
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).Generate(0xc42007e1e0, 0xc4200a8360)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:525 +0x4fb
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).runPlugins(0xc4200aac00, 0xc4200a8360)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1226 +0x59
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).generate(0xc4200aac00, 0xc4200a8360)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1282 +0x377
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).GenerateAllFiles(0xc4200aac00)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1212 +0x259
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command.Generate(0xc4200540c0, 0x6)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command/command.go:131 +0x232
main.main()
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/pkg/cmd/protoc-gen-gogoroach/main.go:91 +0x162
--gogoroach_out: protoc-gen-gogoroach: Plugin failed with status code 2.
panic: protoc-gen-gogo: WARNING: failed finding publicly imported dependency for .cockroach.util.hlc.Timestamp, used in cockroach/pkg/storage/engine/enginepb/mvcc.proto, from cockroach/pkg/util/hlc/timestamp.proto

goroutine 1 [running]:
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).ObjectNamed(0xc4200a9c00, 0xc420291c60, 0x1d, 0x10, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1130 +0x5fa
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).IsMap(0xc4200a9c00, 0xc42029b730, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/helper.go:219 +0x61
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc42029b730, 0xc4201a81b0, 0x2, 0x2, 0xc4204f4628, 0x0, 0x0, 0xb)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:449 +0x5f
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).hasLoop(0xc42007e1e0, 0xc42043b730, 0xc4201a81b0, 0x2, 0x2, 0xc4204f4628, 0x0, 0x0, 0xc4204f4668)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:474 +0x522
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).loops(0xc42007e1e0, 0xc42043b730, 0xc4204003c0, 0xc4200ed100)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:490 +0xb2
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate.(*plugin).Generate(0xc42007e1e0, 0xc4200a6900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/plugin/populate/populate.go:525 +0x4fb
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).runPlugins(0xc4200a9c00, 0xc4200a6900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1226 +0x59
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).generate(0xc4200a9c00, 0xc4200a6900)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1282 +0x377
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).GenerateAllFiles(0xc4200a9c00)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/protoc-gen-gogo/generator/generator.go:1212 +0x259
github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command.Generate(0xc4200540c0, 0x10)
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/vanity/command/command.go:131 +0x232
main.main()
	/Users/tschottdorf/go/src/github.com/cockroachdb/cockroach/pkg/cmd/protoc-gen-gogoroach/main.go:91 +0x162
--gogoroach_out: protoc-gen-gogoroach: Plugin failed with status code 2.
```

--- a/plugin/populate/populate.go
+++ b/plugin/populate/populate.go
@@ -443,7 +443,7 @@ func (p *plugin) GenerateField(file *generator.FileDescriptor, message *generato
 	}
 }
 
-func (p *plugin) hasLoop(field *descriptor.FieldDescriptorProto, visited []*generator.Descriptor, excludes []*generator.Descriptor) *generator.Descriptor {
+func (p *plugin) hasLoop(pkg string, field *descriptor.FieldDescriptorProto, visited []*generator.Descriptor, excludes []*generator.Descriptor) *generator.Descriptor {
 	if field.IsMessage() || p.IsGroup(field) || p.IsMap(field) {
 		var fieldMessage *generator.Descriptor
 		if p.IsMap(field) {
@@ -467,11 +467,11 @@ func (p *plugin) hasLoop(field *descriptor.FieldDescriptorProto, visited []*gene
 				return fieldMessage
 			}
 		}
-		pkg := strings.Split(field.GetTypeName(), ".")[1]
+
 		for _, f := range fieldMessage.Field {
-			if strings.HasPrefix(f.GetTypeName(), "."+pkg+".") {
+			if strings.HasPrefix(f.GetTypeName(), "."+pkg) {
 				visited = append(visited, fieldMessage)
-				loopTo := p.hasLoop(f, visited, excludes)
+				loopTo := p.hasLoop(pkg, f, visited, excludes)
 				if loopTo != nil {
 					return loopTo
 				}
@@ -481,13 +481,13 @@ func (p *plugin) hasLoop(field *descriptor.FieldDescriptorProto, visited []*gene
 	return nil
 }
 
-func (p *plugin) loops(field *descriptor.FieldDescriptorProto, message *generator.Descriptor) int {
+func (p *plugin) loops(pkg string, field *descriptor.FieldDescriptorProto, message *generator.Descriptor) int {
 	//fmt.Fprintf(os.Stderr, "loops %v %v\n", field.GetTypeName(), generator.CamelCaseSlice(message.TypeName()))
 	excludes := []*generator.Descriptor{}
 	loops := 0
 	for {
 		visited := []*generator.Descriptor{}
-		loopTo := p.hasLoop(field, visited, excludes)
+		loopTo := p.hasLoop(pkg, field, visited, excludes)
 		if loopTo == nil {
 			break
 		}
@@ -522,7 +522,7 @@ func (p *plugin) Generate(file *generator.FileDescriptor) {
 		loopLevels := make([]int, len(message.Field))
 		maxLoopLevel := 0
 		for i, field := range message.Field {
-			loopLevels[i] = p.loops(field, message)
+			loopLevels[i] = p.loops(file.GetPackage(), field, message)
 			if loopLevels[i] > maxLoopLevel {
 				maxLoopLevel = loopLevels[i]
 			}

--- a/test/issue270/a/a1.proto
+++ b/test/issue270/a/a1.proto
@@ -1,0 +1,12 @@
+syntax = "proto2";
+
+package issue270.a;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/gogo/protobuf/test/issue270/a/a2.proto";
+
+option (gogoproto.populate_all) = true;
+
+message A1 {
+    optional A2 a2 = 1;
+}

--- a/test/issue270/a/a2.proto
+++ b/test/issue270/a/a2.proto
@@ -1,0 +1,12 @@
+syntax = "proto2";
+
+package issue270.a;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/gogo/protobuf/test/issue270/b/b.proto";
+
+option (gogoproto.populate_all) = true;
+
+message A2 {
+    optional issue270.b.B b = 1;
+}

--- a/test/issue270/b/b.proto
+++ b/test/issue270/b/b.proto
@@ -1,0 +1,6 @@
+syntax = "proto2";
+
+package issue270.b;
+
+message B {
+}

--- a/test/issue270/doc.go
+++ b/test/issue270/doc.go
@@ -1,0 +1,1 @@
+package issue270

--- a/test/issue270/issue270_test.go
+++ b/test/issue270/issue270_test.go
@@ -1,0 +1,51 @@
+// Protocol Buffers for Go with Gadgets
+//
+// Copyright (c) 2013, The GoGo Authors. All rights reserved.
+// http://github.com/gogo/protobuf
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+//     * Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above
+// copyright notice, this list of conditions and the following disclaimer
+// in the documentation and/or other materials provided with the
+// distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+package issue270
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestPopulateWarning(t *testing.T) {
+	cmd := exec.Command("protoc", "--gogo_out=.", "-I=../../../../../:../../protobuf/:.", "a/a1.proto")
+	data, err := cmd.CombinedOutput()
+	dataStr := string(data)
+	t.Logf("received error = %v and output = %v", err, dataStr)
+	if err != nil {
+		t.Error(err)
+	} else if strings.Contains(dataStr, "WARNING") {
+		t.Errorf("Unexpected WARNING: %s", dataStr)
+	}
+	if err = os.Remove("a/a1.pb.go"); err != nil && !os.IsNotExist(err) {
+		t.Error(err)
+	}
+}

